src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { selectedSlug } = useParams();
+
+  const getParent = (name: string | null): Person | undefined => {
+    return name ? people.find(person => person.name === name) : undefined;
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const mother = getParent(person.motherName);
+          const father = getParent(person.fatherName);
```

Comment:
I think will be better to take this logic out and write in a place where you have this function (```getParent```)